packages/components/navbar/src/NavbarAccount/NavbarAccount.tsx

```diff
@@ -51,11 +54,12 @@ function _NavbarAccount(
           className={cx(styles.root, className)}
           testId={testId}
         >
-          <img
+          {/* <img
```

Comment:
If this block of code isn't required, shall we remove it altogether? Alternatively, is there a specific reason for keeping it commented out?

Reply:
Good catch, thanks!